docs/src/main/asciidoc/funqy-http.adoc

```diff
@@ -97,15 +107,198 @@ curl "http://localhost:8080/double" \
 -d '2'
 ----
 
-== Maven Dependencies
+== GET Query Parameter Mapping
 
-To write Funqy HTTP functions, simply include the `quarkus-funqy-http` dependency into your Quarkus `pom.xml` file:
+For GET requests, the Funqy HTTP Binding also has a query parameter mapping for function input parameters.
+Only bean style classes and `java.util.Map` can be used for your input parameter.  For bean style
+classes, query parameter names are mapped to properties on the bean class.  Here's an example of a simple
+`Map`:
 
-[source, xml]
+[source, java]
 ----
-<dependency>
-    <groupId>io.quarkus</groupId>
-    <artifactId>quarkus-funqy-http</artifactId>
-</dependency>
+@Funq
+public String hello(Map<String, Integer> map) {
+...
+}
+----
+
+Key values must be a primitive type or `String`.  Values can be primitives, `String`, `OffsetDateTime` or a complex
```

Comment:
Why `OffsetDateTime` and not `Date` of the other date types inside java.time ?

Reply:
Its parse() method is a specific format ISO-8601 where Date requires a DateFormat.  Using offset for date time queries is also a better data structure as dates are usually stored by absolute time.